Material_TimePicker/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@ohos/material_timepicker",
```

Comment:
Change the folder name from Material_TimePicker to MaterialTimePicker

Reply:
done